internal/action/setup.go

```diff
@@ -137,6 +138,10 @@ func (s *Action) initGenerateIdentity(ctx context.Context, crypto backend.Crypto
 		}
 
 		email, err = termio.AskForString(ctx, "📧 What is your email?", email)
+
+		if strings.TrimSpace(email) == "" {
```

Comment:
nit: I think checking for the email string should happen after error handling in the next block, i.e. please move these lines below line 147.

Reply:
Done @dominikschulz

Reply:
@dominikschulz  Can you please resolve this conversation?